bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,285 @@ class CoffeeLintBear:
+                        braces_spacing_width: int=0,
+                        spacing_in_empty_braces: int=0,
+                        class_naming_convention: str='',
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        enforce_newline_at_EOF: bool=False,
+                        use_spaces: bool=True,
+                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                        newlines_after_classes: bool=False,
+                        number_of_newlines_after_classes: int=2,
+                        no_backticks: bool=True,
+                        no_implicit_braces: bool=False,
+                        no_implicit_parentheses: bool=False,
+                        no_interpolation_in_single_quotes: bool=True,
+                        no_stand_alone_at: bool=True,
+                        disable_throwing_strings: bool=True,
+                        allow_trailing_semicolons: bool=False,
+                        allow_trailing_whitespaces: bool=False,
+                        no_unnecessary_double_quotes: bool=False,
+                        use_english_operator: bool=False,
+                        spaces_around_operators: bool=False,
+                        space_after_comma: bool=True,
+                        cyclomatic_complexity: int=10,
+                        prevent_duplicate_keys: bool=True,
+                        consistent_line_endings: bool=False,
+                        consistent_line_endings_style: str='unix',
+                        no_this: bool=False,
+                        no_decr_or_incrementation_operators: bool=False,
+                        no_empty_parameter_list: bool=False,
+                        no_empty_functions: bool=False,
+                        enforce_parentheses_on_non_empty_constructors: bool=True
+                        ):
+        """
+        :param max_line_length:
+            Maximum number of characters per line.
+        :param max_line_length_affects_comments:
+            Determines if ``max_line_length`` should also affects comments or
+            not.
+        :param space_before_and_after_arrow:
+            Determines if spaces should be used before and/or after the arrow.
+        :param check_braces_spacing:
+            Checks if the proper spacing is used inside curly braces.
+        :param braces_spacing_width:
+            Determines the number of blank spaces after the opening ``{`` and
+            before the closing brace ``}`` given that there is something within
+            the braces.
+        :param spacing_in_empty_braces:
+            Determines the number of blank spaces after the opening ``{`` and
+            before the closing brace ``}`` given an empty content.
+        :param class_naming_convention:
+            Checks whether the classes name should be in camel-case or not.
```

Comment:
I would highly suggest to rename this parameter:
```
class_naming_camel_case
```
or something like that, since setting this flag enforces a specific naming convention. Also make this a bool then. Or if possible you can support multiple ones.